swampmachine-multimedia/src/main/java/net/kiberion/audio/AudioService.java

```diff
@@ -23,20 +20,27 @@
     @Autowired
     public void setMixer(Mixer mixer) {
         this.mixer = mixer;
-        musicChannel = mixer.newChannel(MUSIC_CHANNEL_ID);
-        soundChannel = mixer.newChannel(SOUND_CHANNEL_ID);
     }
     
     @EventListener
     public void playSound (PlaySoundEvent event) {
         log.info("Play sound");
-        soundChannel.playSound(soundRegistry.getSoundEntries().get(event.getSoundId()));
+        mixer.getChannel("sound").playSound(soundRegistry.getSoundEntries().get(event.getSoundId()));
```

Comment:
yeah, hardcoded inline string is so much better than a constant. Make it a public static final, but keep it const.